src/components/LoginHint/index.js

```diff
@@ -1,7 +1,9 @@
 import React, { Component } from 'react';
 import { routerRedux } from 'dva/router';
 import { connect } from 'dva';
-import { Title, Button } from '@patternfly/react-core';
+import { Title, Button, Banner, Flex, FlexItem } from '@patternfly/react-core';
```

Comment:
These changes seem unrelated to the PR. Did you mean to include changes to update the login hint UI/UX?

Reply:
Yeah, so we had a discussion in the design sync to add the login banner rather than login hint